templates/pro.html

```diff
@@ -0,0 +1,566 @@
+          </div>        
+          <div class="col-9">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/c53664cd-Security_updates.svg",
+                        alt="",
+                        width="251",
+                        height="150",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+            <p>開発者の活躍をサポート。自分のペースでアップグレードしつつ、スタック全体のセキュリティを確保。</p>
+            <p>ESMは、危険、高、一部の中レベルのCVEに対する10年間の脆弱性管理で、Ubuntuを使用するすべてのソフトウェアパッケージを守ります。</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4">
+        <div class="row">
+          <hr class="is-muted"/>
+          <div class="col-3">
+            <strong>ライブカーネルパッチ</strong>
+          </div>        
+          <div class="col-9">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/c87f3151-Update.svg",
+                        alt="",
+                        width="155",
+                        height="150",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+            <p>危険または緊急度の高いカーネル脆弱性にパッチを適用し、ダウンタイムと計画外の再起動を抑えます。</p>
+            <p>IT従事者の64%は年間100時間以上を予定外の仕事に費やしています。ライブパッチ機能により、予定されたメンテナンス時間外にマシンを再起動することなくCVEにパッチを適用できます。</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4">
+        <div class="row">
+          <hr class="is-muted"/>
+          <div class="col-3">
+            <strong>コンプライアンスとハードニング</strong>
+          </div>        
+          <div class="col-9">
+            <div class="p-image-container--16-9 is-highlighted">
+              {{ image(url="https://assets.ubuntu.com/v1/1d5d8d10-compliance-icon.svg",
+                        alt="",
+                        width="150",
+                        height="150",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+            <p>CIS、DISA-STIG、FIPS 140、Common Criteriaなど、ハードニングとコンプライアンスのプロファイルに対応する認定済みの自動化ツールで業務のリスクを軽減しましょう。</p>
+            <p>Ubuntu Proは、オンプレミスやパブリッククラウド（AWS、Azure、GCPなど）でコンプライアンスを維持するツールを提供します。</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom is-muted" />
+      <div class="col">
+        <h3 class="p-heading--2">Ubuntu Proのアドオン</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4">
```

Comment:
Lmk if the pastebin is helpful here as well, should be the same layout

Reply:
Yep, I applied the same layout here